### PR TITLE
Write API docs

### PR DESCRIPTION
**Pull request by vweevers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,39 +94,70 @@ See [`memdown`](https://github.com/Level/memdown/) if you are looking for a comp
...
+
+### `db.open([options, ]callback)`
+### `db.close(callback)`
+### `db.get(key[, options], callback)`
+### `db.put(key, value[, options], callback)`
+### `db.del(key[, options], callback)`
+### `db.batch(operations[, options], callback)`
+### `chainedBatch = db.batch()`
+
+If `batch()` is called without arguments ~~or with only an options object~~ then it will return a `chainedBatch` object with chainable methods.
```
Comment (vweevers) [inline README.md]: As far as I can see, "or with only an options object" is (no longer?) true.

Comment (vweevers) [reply README.md]: => #254

File: README.md [hunk 2]
```diff
@@ -175,18 +187,24 @@ An iterator keeps track of when a `next()` is in progress and when an `end()` ha
 
 ## Private API for implementors
 
-Each of these methods will receive exactly the number and order of arguments described. Optional arguments will receive sensible defaults.
+Each of these methods will receive exactly the number and order of arguments described. Optional arguments will receive sensible defaults. All callbacks are error-first and must be asynchronous. If an operation within your implementation is synchronous, be sure to call the callback in a next tick using `process.nextTick(callback, ..)`, `setImmediate` or some other means of micro- or macrotask scheduling.
```
Comment (ralphtheninja) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -175,18 +187,24 @@ An iterator keeps track of when a `next()` is in progress and when an `end()` ha
...
 
-### `db = constructor(location)`
+### `db = AbstractLevelDOWN(location)`
 
-Currently, the `abstract-leveldown` constructor expects a location argument and throws if one isn't given. If your implementation doesn't have a `location`, pass an empty string (`''`).
+The constructor expects a location argument and throws if one isn't given. If your implementation doesn't have a `location`, pass an empty string (`''`).
```
Comment (ralphtheninja) [inline README.md]: @vweevers I know you're documenting the current state, but maybe later we could change this so location defaults to `''`?

**Comment by vweevers** [comment]
Only `_serialize*` is left. After this I'll do a pass on the `leveldown` docs too.

**Pull request merged**